test/unit/math/mix/scal/fun/F32_test.cpp

```diff
@@ -1,4 +1,7 @@
 #include <stan/math/mix/scal.hpp>
+#include <stan/math/mix/mat.hpp>
```

Comment:
1. there are 3 includes of `stan/math/mix/mat.hpp`
2. this test should not include `mat.hpp`. why does it need `mat.hpp`?